platform/index.md

```diff
@@ -1,18 +1,70 @@
 ---
-title: Platform
-summary: Platform level index for articles, guidance and documentation
+title: Particular Service Platform Overview
+summary: A short overview of Particular Platform
+tags: 
+- Particular Service Platform, NServiceBus, ServiceMatrix, ServiceControl, ServiceInsight, ServicePulse
 redirects:
  - nservicebus/preparing-your-machine-to-run-nservicebus
 ---
 
-### Platform
+# Particular Service Platform
 
 - [Getting started with the Particular Service Platform](getting-started-with-particular-service-platform.md)
 - [Releases and Release Notes](release-notes.md)
 - [Platform Installer](installer)
 
-### Applications
+Everything started with a simple <cite>wrapper library around System Messaging API [1]</cite>. Few years later, it was shared with wider public in the form of an open-source project. Following the popular .NET convention it was called simply NServiceBus. The project gained popularity and soon it became clear that it would be difficult to keep meeting user demands without a dedicated team of developers. That is how the Particular Software company came to life. 
 
+Messaging is fantastic for building loosely coupled, scalable and reliable systems. However, it also has challenges. Two most common pains are having visibility of what is happening in the system and monitoring. In the beginning, teams using NServiceBus relied on generic monitoring applications or they built their own. After a while it became obvious that they could benefit from tools tailored to specific needs of NServiceBus. The other challenge is to understand the principles of building distributed systems and applying them in practice. This might be especially difficult for less experienced teams or developers that worked in a different way for many years. 
```

Comment:
Fair point, the intention was to somehow explain where SM comes from, but it sounds aweful indeed.